golden/clr-core.d.ts

```diff
@@ -1,16 +1,30 @@
+export declare const angleDoubleIcon: IconShapeTuple;
+
+export declare const angleIcon: IconShapeTuple;
+
+export declare const arrowIcon: IconShapeTuple;
+
+export declare const barsIcon: IconShapeTuple;
+
+export declare const bellIcon: IconShapeTuple;
+
+export declare const calendarIcon: IconShapeTuple;
+
+export declare const checkCircleIcon: IconShapeTuple;
+
+export declare const checkIcon: IconShapeTuple;
+
 export declare class ClarityIcons {
-    static readonly registry: {
-        [x: string]: string;
-    };
+    static readonly registry: IconRegistrySources;
     static add(shapes: IconShapeSources): void;
-    static addCollection(collection: IconCollection): void;
-    static addIcon(shape: IconShapeTuple): void;
-    static addIcons(shapes: IconShapeTuple[]): void;
-    static alias(aliases: IconAlias): void;
-    static get(shapeName?: string): string | IconShapeCollection;
-    static has(shapeName: string): boolean;
+    static addAliases(...aliases: IconAlias[]): void;
+    static addIcons(...shapes: IconShapeTuple[]): void;
+    static alias(alias: IconAliasLegacyObject): void;
+    static get(shapeName?: string): string | IconRegistrySources;
 }
 
+export declare const cloudIcon: IconShapeTuple;
+
 export interface ClrCommonStrings {
```

Comment:
I wish these interfaces / classes weren't interspersed between the list of icon `const`s. Is this because they are automatically alphabetized and ordered?

Reply:
Ah this is the `d.ts` file. Ignore my comment. 😁